stable/redis/values-production.yaml

```diff
@@ -0,0 +1,230 @@
+## Bitnami Redis image version
+## ref: https://hub.docker.com/r/bitnami/redis/tags/
+##
+image: bitnami/redis
+imageTag: 4.0.9-r0
+
+## Specify a imagePullPolicy
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+imagePullPolicy: IfNotPresent
+
+## Image credentials
+# imageCredentials:
+  # registry: docker.io
+  # username: javsalgar
+  # password: password
+
+## Cluster settings
+cluster:
+  enabled: true
```

Comment:
enabled by default?

Reply:
In a production environment you wouldn't want a single node redis, but a cluster, right?

Reply:
aha.. i did not notice this was `values-production.yaml` :slightly_smiling_face: